cmd/kubeadm/app/apis/bootstraptoken/v1/defaults.go

```diff
@@ -25,7 +25,7 @@ import (
 
 const (
 	// DefaultTokenDuration specifies the default amount of time that a bootstrap token will be valid
-	// Default behaviour is 24 hours
+	// Default behavior is 24 hours
```

Comment:
Behaviour and behavior are two spellings of the same noun. The spelling depends on the type of English. In British English, you use “behaviour.” In American English, you use “behavior.”

I don't think this need to be correct.

Reply:
we tend to use USA english